src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit,
+} from '@angular/core';
+import { Validators } from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+import { FormBuilder } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { Store } from '@ngrx/store';
+import { TranslateService } from '@ngx-translate/core';
+import { getPoolStatusLabels, PoolStatus } from 'app/enums/pool-status.enum';
+import { BootPoolState } from 'app/interfaces/boot-pool-state.interface';
+import { EntityUtils } from 'app/modules/entity/utils';
+import { FormErrorHandlerService } from 'app/modules/ix-forms/services/form-error-handler.service';
+import { AppLoaderService, DialogService, WebSocketService } from 'app/services';
+import { AppState } from 'app/store';
+import { waitForAdvancedConfig } from 'app/store/system-config/system-config.selectors';
+
+@UntilDestroy()
+@Component({
+  templateUrl: './bootenv-stats-dialog.component.html',
+  styleUrls: ['./bootenv-stats-dialog.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BootenvStatsDialogComponent implements OnInit {
+  form = this.fb.group({
+    interval: [null as number, [Validators.required, Validators.min(1)]],
```

Comment:
`interval: [null as number, [Validators.required, numberValidator(), Validators.min(1)]],`

Reply:
Fixed the translations, but I don't see why I need to have numberValidator(). We've agreed that `type` in ix-input should provide validation for number.

Reply:
Sorry, You are right. `ix-input` has own validator for type.